src/anonymous-credentials/schema.ts

```diff
@@ -626,6 +631,8 @@ export class CredentialSchema extends Versioned {
         return { type: ValueType.PositiveInteger };
       case CredentialSchema.BOOLEAN_TYPE:
         return { type: ValueType.PositiveInteger };
+      case CredentialSchema.DATE_TYPE:
+        return { type: ValueType.Integer, minimum: value['minimum'] };
```

Comment:
`ValueType` should be `PositiveInteger`. And you shouldn't need `minimum`.

Reply:
incorrect, dates can be negative